### DISCUSSION TITLE
Do you plan to support C++ ?

### DISCUSSION THREAD

**Discussion by hardBSDk** [immunant/c2rust#1085] [Q&A]
C++ to Rust translation.

**Comment by thedataking** [comment]
There are no current plans to support C++ (or more realistically, a subset thereof). Having C++ support would be valuable but alsos incredibly difficult as it is a much larger language than C. We are currently focusing our limited resources on supporting C better, e.g. by analyzing how C pointers can be mapped to safe Rust types such as slices and arrays as well as maintenance and bugfixes.
